muse_for_anything/db/models/ontology_objects.py

```diff
@@ -23,9 +23,9 @@ class OntologyObjectType(MODEL, IdMixin, NameDescriptionMixin, ChangesMixin):
 
     __tablename__ = "Type"
 
-    namespace_id: Mapped[int] = mapped_column(ForeignKey("Namespace.id"), nullable=False)
+    namespace_id: Mapped[int] = mapped_column(ForeignKey(Namespace.id), nullable=False)
     current_version_id: Mapped[int] = mapped_column(
-        ForeignKey("TypeVesion.id"), nullable=True
+        ForeignKey("TypeVersion.id"), nullable=True
```

Comment:
How did this not result in an error already? Another thing to be checked for a db migration...

Reply:
Same as before, this typo is on me and caused an error during testing (only before, other errors came before)